stories/Header/Headers.stories.mdx

```diff
@@ -1,57 +1,113 @@
-* Una parte dedicata alla navigazione “nav header“, visibile su schermi di grandi dimensioni ed accessibile attraverso il classico bottone di tipo “burger menu” per dispositivi mobili.
+- Un cosiddetto “slim header”, una sottile fascia dello stesso colore o, preferibilmente, di colore lievemente più scuro rispetto al tema principale del sito, che mostra alcuni link funzionali con impatto globale o esterno al sito stesso.
+- Una parte centrale “center header“ che identifica in modo chiaro il sito attraverso logo, testo e social, e può contenere un link per effettuare ricerche sul sito.
+- Una parte dedicata alla navigazione “nav header“, visibile su schermi di grandi dimensioni ed accessibile attraverso il classico bottone di tipo “burger menu” per dispositivi mobili.
 
 ## Slim header
 
-Lo *Slim header* header mostra un’intestazione, solitamente con riferimento all’ente di appartenenza del progetto o riferimenti utili, oltre ad un eventuale menu per il cambio lingua e l’accesso ad area riservata. Il cambio lingua è gestito con il componente dropdown.
+Lo _Slim header_ header mostra un’intestazione, solitamente con riferimento all’ente di appartenenza del progetto o riferimenti utili, oltre ad un eventuale menu per il cambio lingua e l’accesso ad area riservata. Il cambio lingua è gestito con il componente dropdown.
 
 <Canvas>
-    <Story id="componenti-header--header-slim" />
+  <Story id='componenti-header--header-slim' />
+</Canvas>
+
+### Right action con bottone full responsive
+
+Per trasformare il bottone di action situato nella `HeaderRightZone` e renderlo full-responsive è sufficiente applicare la prop `size="full"` al link/bottone.
+
+Il modificatore `"full"` è disponibile anche con il tema chiaro attivato da .theme-light.
+
+<Canvas>
+  <Story id='componenti-header--slim-header-with-full-button' />
 </Canvas>
 
 ### Versione chiara
 
 Per cambiare tema all’header slim è sufficiente aggiungere il modificatore `theme="light"` al componente `Header` di tipo `slim`.
 
-### Right action con bottone full responsive
+<Canvas>
+  <Story id='componenti-header--header-slim-light' />
+</Canvas>
 
-Per trasformare il bottone di action situato nella `HeaderRightZone` e renderlo full-responsive è sufficiente applicare la prop `size="full"` al link/bottone.
+## Header Centrale
 
-Il modificatore `"full"` è disponibile anche con il tema chiaro attivato da .theme-light.
+<Canvas>
+  <Story id='componenti-header--header-center' />
+</Canvas>
+
+### Versione stretta
+
+<Canvas>
+  <Story id='componenti-header--header-center-narrow' />
+</Canvas>
+
+### Versione chiara
+
+<Canvas>
+  <Story id='componenti-header--header-center-narrow-light' />
+</Canvas>
+
+## Header Nav
+
+<Canvas>
+  <Story id='componenti-header--header-nav' />
+</Canvas>
+
+### Versione chiara
 
 <Canvas>
-    <Story id="componenti-header--slim-header-with-full-button" />
+  <Story id='componenti-header--header-nav-light' />
 </Canvas>
 
+## Header Completa
```

Comment:
Just before this story I think an example is missing: the Header with secondary links.

<img width="916" alt="Screenshot 2021-05-25 at 08 39 42" src="https://user-images.githubusercontent.com/924948/119450824-c4a48f00-bd34-11eb-8525-f281818c4b9f.png">


Here's the example reference from the BI doc: https://italia.github.io/bootstrap-italia/docs/menu-di-navigazione/header/#navigazione-secondaria

Reply:
Added Header Nav secondary example :+1: